<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finance</class>
 <widget class="QWidget" name="Finance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>金融理财</string>
  </property>
  <widget class="QLabel" name="labelIncome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>收益率计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIncomeSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIncomeTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资期限：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIncomeDest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>到期金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIncomeDest_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资收益：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputSrc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string extracomment="0.00">0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>day</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputDest">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputRate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuli">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>复利计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuliSrc">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuliTime">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资期限：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuliRate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年收率益：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFuliDest">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资收益：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFuliSrc">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFuliTime">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFuliRate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFuliDest">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalcRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalcFuli">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
